litmus-portal/frontend/src/containers/layouts/Header/ProfileDropdownItems.tsx

```diff
@@ -51,6 +55,12 @@ const ProfileInfoDropdownItems: React.FC<ProfileInfoDropdownItemProps> = ({
   const initials = nameSplit[1]
     ? userAvatar(name, false)
     : userAvatar(name, true);
+  // Query to get user details
+  const { data, loading } = useQuery<
+    CurrentUserDetails,
+    CurrentUserDedtailsVars
+  >(GET_USER, { variables: { username } });
+  const projects: Project[] = data?.getUser.projects ?? [];
```

Comment:
```suggestion
  const projects = data?.getUser.projects ?? [];
```
no need to type this....it's already inferred by the ts compiler